internal/test_unit/xml/unit_tc_pairs.xml

```diff
@@ -182,4 +182,27 @@
     </output>
   </test>
 
+  <test name="tc_pairs_DIAGNOSTICS">
+  <exec>&MET_BIN;/tc_pairs</exec>
+    <env>
+      <pair><name>STORM_ID</name> <value>"AL092022"</value></pair>
+      <pair><name>INIT_BEG</name> <value>"20220926_00"</value></pair>
+      <pair><name>INIT_END</name> <value>"20220926_18"</value></pair>
+      <pair><name>DIAG_NAME</name> <value></value></pair>
```

Comment:
It looks like there is no value set for DIAG_NAME that is used to set diag_name in the TCPairsConfig_DIAGNOSTICS file. It would be useful to see an example of what is a valid value for adding this functionality to the METplus wrapper. Is it a string surrounded by quotation marks? Adding a test where a value is set here would be useful to ensure that it works as expected.

Reply:
@georgemccabe great point. When `diag_name` is empty, ALL diags from the input are passed to the output. I updated the test to provide a specific list with some names for the TCDIAG input and other names for the LSDIAG_RT input. That's a better test. I did manually test to confirm that an empty list still results in all being written, and it does.

Thanks for catching that.

Reply:
FYI, you can ignore the ascii2nc diff. Those are from a recent PR that corrected an recent bug in ascii2nc. I just haven't updated develop-ref yet. Was hoping to include diffs from a second open PR in that update step.